imageio/config/extensions.py

```diff
@@ -596,7 +596,7 @@ def reset(self):
     ),
     FileExtension(
         extension=".nef",
-        priority=["RAW-FI"],
+        priority=["rawpy", "RAW-FI"],
```

Comment:
```suggestion
        priority=["RAW-FI", "rawpy"],
```

Since this is a new plugin, I would like to start with it having a low priority while we test for bugs/issues. Once it has been around for a while it makes sense to have it become the default choice for supported raw formats.

Reply:
Thanks! I will make it a low priority when the all implementations are done. I have kept it so far in this way to complement tests.